### PR TITLE
Kafka Integration Docs

### PR DESCRIPTION
**Pull request by kraftp**

### CODE REVIEW

File: docs/tutorials/kafka-integration.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+sidebar_position: 18
+title: Integrating with Kafka
+description: Learn how to integrate DBOS and Kafka.
+---
+
+In this guide, you'll learn how to use DBOS transactions and workflows to process Kafka messages with exactly-once semantics.
+
+First, define your transaction or workflow. It must take in the Kafka topic, partition, and message as inputs:
+
+```javascript
```
Comment (chuck-dbos) [inline docs/tutorials/kafka-integration.md:11]: Is this the best for ts?

File: docs/tutorials/kafka-integration.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+  @KafkaConsume("example-topic")
+  @Workflow()
+  static async kafkaWorkflow(ctxt: WorkflowContext, topic: string, partition: number, message: KafkaMessage) {
+    ctxt.logger.info(`Message received: ${message.value?.toString()}`)
+  }
+}
+```
+
+If you need more control, you can pass detailed configurations to both the `@Kafka` and `@KafkaConsume` decorators.
```
Comment (chuck-dbos) [inline docs/tutorials/kafka-integration.md:42]: We should add these (`Kafka` and `KafkaConsume`) to the import?

File: docs/tutorials/kafka-integration.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  }
+}
+```
+
+Then, annotate your method with a [`@KafkaConsume`](../api-reference/decorators.md#kafka-consume) decorator specifiying which topic to consume from.
+Additionally, annotate your class with a [`@Kafka`](../api-reference/decorators.md#kafka) decorator defining which brokers to connect to.
+DBOS invokes your method exactly-once for each message sent to the topic.
+
+```javascript
+import { KafkaConfig, KafkaMessage} from "kafkajs";
```
Comment (chuck-dbos) [inline docs/tutorials/kafka-integration.md:25]: Everybody already knows to do this, but for completeness `npm install kafkajs`.

File: docs/tutorials/kafka-integration.md (hunk 2 above)
Comment (chuck-dbos) [reply docs/tutorials/kafka-integration.md:42]: Might as well document the full import needed:
`import { Workflow, WorkflowContext, Kafka, KafkaConsume } from '@dbos-inc/dbos-sdk';`

File: docs/tutorials/kafka-integration.md (hunk 1 above)
Comment (kraftp) [reply docs/tutorials/kafka-integration.md:11]: I also tried `typescript`, but it didn't work very well.

**Review by chuck-dbos — APPROVED** [review]

**Pull request merged**